spec/models/collection_spec.rb

```diff
@@ -147,6 +155,10 @@
     end
     it {is_expected.to validate_presence_of(:unit)}
     it {is_expected.to validate_inclusion_of(:unit).in_array(Admin::Collection.units)}
+    it { is_expected.to allow_value('collection@example.com').for(:contact_email) }
+    it { is_expected.not_to allow_value('collection@').for(:contact_email) }
+    it { is_expected.to allow_value('https://collection.example.com').for(:website_url) }
+    it { is_expected.not_to allow_value('collection.example.com').for(:website_url) }
```

Comment:
This may be a little a little too strict in practice as people aren't used to typing out the "https://" part, but I guess it makes for a safer validator.